spec/routing/rhsm/consumers_routing_spec.rb

```diff
@@ -10,29 +10,29 @@ module Katello
+        ({:controller => proxies_controller, :action => "get", :id => "1"}).must_recognize(:method => "get", :path => "/rhsm/consumers/1/certificates")
+        ({:controller => proxies_controller, :action => "get", :id => "1"}).must_recognize(:method => "get", :path => "/rhsm/consumers/1/certificates/serials")
+        ({:controller => proxies_controller, :action => "get", :id => "1"}).must_recognize(:method => "get", :path => "/rhsm/consumers/1/entitlements")
+        ({:controller => proxies_controller, :action => "post", :id => "1"}).must_recognize(:method => "post", :path => "/rhsm/consumers/1/entitlements")
+        ({:controller => proxies_controller, :action => "delete", :id => "1"}).must_recognize(:method => "delete", :path => "/rhsm/consumers/1/entitlements")
+        ({:controller => proxies_controller, :action => "delete", :consumer_id => "1", :id => "1"}).must_recognize(:method => "delete", :path => "/rhsm/consumers/1/certificates/1")
+        ({:controller => proxies_controller, :action => "get"}).must_recognize(:method => "get", :path => "/rhsm/pools")
+        ({:controller => proxies_controller, :action => "get", :id => "1"}).must_recognize(:method => "get", :path => "/rhsm/entitlements/1")
+        ({:controller => proxies_controller, :action => "post"}).must_recognize(:method => "post", :path => "/rhsm/subscriptions")
+        {:controller => proxies_controller, :action => "upload_package_profile", :id => "1"}.must_recognize(:method => "put", :path => "/rhsm/consumers/1/profile/")
+        {:controller => proxies_controller, :action => "get", :id => "1"}.must_recognize(:method => "get", :path => "/rhsm/consumers/1/guestids/")
+        {:controller => proxies_controller, :action => "get", :id => "1", :guest_id => "1"}.must_recognize(:method => "get", :path => "/rhsm/consumers/1/guestids/1")
+        {:controller => proxies_controller, :action => "put", :id => "1"}.must_recognize(:method => "put", :path => "/rhsm/consumers/1/guestids/")
+        {:controller => proxies_controller, :action => "put", :id => "1", :guest_id => "1"}.must_recognize(:method => "put", :path => "/rhsm/consumers/1/guestids/1")
+        {:controller => proxies_controller, :action => "delete", :id => "1", :guest_id => "1"}.must_recognize(:method => "delete", :path => "/rhsm/consumers/1/guestids/1")
+        {:controller => proxies_controller, :action => "get", :id => "1"}.must_recognize(:method => "get", :path => "/rhsm/consumers/1/content_overrides/")
+        {:controller => proxies_controller, :action => "put", :id => "1"}.must_recognize(:method => "put", :path => "/rhsm/consumers/1/content_overrides/")
+        {:controller => proxies_controller, :action => "delete", :id => "1"}.must_recognize(:method => "delete", :path => "/rhsm/consumers/1/content_overrides/")
+        {:controller => proxies_controller, :action => "available_releases", :id => "1"}.must_recognize(:method => "get", :path => "/rhsm/consumers/1/available_releases")
```

Comment:
just for clarity, these were changed in the rails 4 branch, I'm changing them back to match [master](https://github.com/Katello/katello/blob/master/spec/routing/rhsm/consumers_routing_spec.rb)